packages/editor/src/components/post-featured-image/index.js

```diff
@@ -29,49 +30,63 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 	const postLabel = get( postType, [ 'labels' ], {} );
 	const instructions = <p>{ __( 'To edit the featured image, you need permission to upload media.' ) }</p>;
 
-	let mediaWidth, mediaHeight, mediaSourceUrl;
+	let mediaWidth, mediaHeight, mediaSourceUrl, mediaAltText, imageFilename;
 	if ( media ) {
 		const mediaSize = applyFilters( 'editor.PostFeaturedImage.imageSize', 'post-thumbnail', media.id, currentPostId );
 		if ( has( media, [ 'media_details', 'sizes', mediaSize ] ) ) {
 			mediaWidth = media.media_details.sizes[ mediaSize ].width;
 			mediaHeight = media.media_details.sizes[ mediaSize ].height;
 			mediaSourceUrl = media.media_details.sizes[ mediaSize ].source_url;
+			mediaAltText = media.media_details.sizes[ mediaSize ].alt_text;
```

Comment:
I'm not sure I understand how the `PostFeaturedImage`  works (@gziolo is this still working?), however seems to me `alt_text` is never a property of `media.media_details.sizes[ mediaSize ]` and instead it's always `media.alt_text`. If I'm correct, this will fail and will use the wrong description for images that actually do have an alt text.